Misc Security-related Fixes

Lorwp opened the pull request:
- Allows Stun Revolvers to be put in Security Belts
- Implements Master At Arms Holobadge Box (Fixes #15989)
- Adds Security Messenger bags to Security Lockers (Does NOT help #15927, that’s for the people who aren’t shamelessly copying code)
- Adds Work Gloves to ~~NT Guard~~ and Solgov Security ~~Standard Spawn~~ Lockers

Commit:
Misc Security-related Fixes

- Allows Stun Revolvers to be put in Security Belts
- Implements Master At Arms Holobadge Box (Fixes #15989)
- Adds Security Messenger bags to Security Lockers (Does NOT Help #15927, that’s for the people who aren’t shamelessly copying code)
- Adds Work Gloves to NT Guard and Solgov Security Standard Spawn

PsiOmegaDelta commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -100,6 +100,20 @@
+		new /obj/item/clothing/accessory/badge/security(src)
+		new /obj/item/clothing/accessory/badge/security(src)
+		..()
+		return
```
Relative pathing is evil and verboten. You can also write this as:
```
/obj/item/weapon/storage/box/large/holobadge/solgov
	name = "holobadge box"
	desc = "A box claiming to contain holobadges, this one has 'Master at Arms' written on it in fine print."
	startswith = list(/obj/item/clothing/accessory/badge/security = 6)
```

PsiOmegaDelta commented:
Also, this will close #15927 on merge because the word "fix" comes before the number, even if the word "not" comes before that.

Edit: Now corrected.

Commit:
Banish Thee Relative Pathing

Techhead0 commented on maps/torch/job/outfits.dm:
```diff
@@ -674,6 +678,7 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 	uniform = /obj/item/clothing/under/rank/guard
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/soft/sec/corp/guard
+	gloves = /obj/item/clothing/gloves/thick
```
Also, I don't think the NT security guard should get gloves at all.

Lorwp replied:
I'll remove these, as they are not as vital to the Security guard, as they (almost) never work with Forensics directly.

Techhead0 commented on maps/torch/job/outfits.dm:
```diff
@@ -314,6 +316,7 @@ Keeping them simple for now, just spawning with basic EC uniforms, and pretty mu
 	uniform = /obj/item/clothing/under/utility/expeditionary/security
 	shoes = /obj/item/clothing/shoes/dress
 	head = /obj/item/clothing/head/soft/sol/expedition
+	gloves = /obj/item/clothing/gloves/thick
```
Forensic tech could probably use `/obj/item/clothing/gloves/forensic`, like they have in their lockers.

Techhead0 reviewed (commented):
If you're giving gloves to all of security, you can do it in the base `/decl/hierarchy/outfit/job/torch/crew/security` type instead of doing the brig officer, forensic tech, and master at arms separately.

Techhead0 commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -100,6 +100,12 @@
 		return
 
 
+/obj/item/weapon/storage/box/large/holobadge/solgov
```
Instead of making a large box to fit solgov badges, why not just fix the holobadge item size?

Lorwp replied:
~~i'm not completely sure how to do this in any case, as there are no references to object size in `badges.dm`which i can see. If the size is defined in another file, i wouldn't know.~~

Lorwp replied:
Done in 	11879fa

Raptor1628 commented:
I tried to avoid giving people gloves on spawn. They're easily available in the loadout, just get them from there.

Lorwp commented:
The reasoning for giving Security gloves on spawn is that it makes the Forensic's tech's job much easier, as i would assume it would be common sense to wear them to not contaminate a Crime Scene.

If you do not wish them to spawn with them, however, i can easily just place a set of Optional Work Gloves in their lockers.

Commit:
#15990 Feedback

- Neater code in outfits.dm for Security Gloves
- MAA Holobadge size smaller, made they’re in Smaller in turn.

DeeUnderscore commented on code/modules/clothing/under/accessories/badges.dm:
```diff
@@ -100,6 +100,13 @@
+	name = "holobadge box"
+	desc = "A box claiming to contain holobadges, this one has 'Master at Arms' written on it in fine print."
+	startswith = list(/obj/item/clothing/accessory/badge/security = 4)
+	w_class = ITEM_SIZE_TINY
```
Making the *box* a tiny item?

Lorwp replied:
Bluespace box, obviously. (and a major mistake)

Raptor1628 commented:
If it's too much effort to select gloves from spawn, feel free to add them to the lockers, but I'm trying to keep people spawning with the bare minimum equipment, forcing them to go equip together.

Lorwp commented:
Closing this, this is a fine mess, will reopen later 👌
